utils/binv.f90

```diff
@@ -43,18 +49,18 @@ PROGRAM BINV
       NINV(2,ISUB) = NINV(2,ISUB)+NMSUB(LUNBF)
       NINV(3,ISUB) = NINV(3,ISUB)+NMBYT(LUNBF)
  
-c     IOFF = 1
-c     CALL STATUS(LUNBF,LUN,IL,IM)
-c     call ufbcnt(lunbf,irec,isub)
-c     DO I=1,NMSUB(LUNBF)
-c     NBYS = IUPB(MBAY(1,LUN),MBYT(LUN)+IOFF,16)
-c     print*,SUBSET,' m#',irec,' subt#',i,nbys
-c     IOFF = IOFF+NBYS
-c     ENDDO
+!     IOFF = 1
```

Comment:
Is there any particular reason why we're keeping these 8 commented-out lines of code?

Reply:
None. It can be cleaned up